addons/website_event_questions/models/event_registration.py

```diff
@@ -28,3 +28,10 @@ class EventRegistrationAnswer(models.Model):
     _sql_constraints = [
         ('value_check', "CHECK(value_answer_id IS NOT NULL OR COALESCE(value_text_box, '') <> '')", "There must be a suggested value or a text value.")
     ]
+
+    # for displaying selected answers by attendees in attendees list view
+    def name_get(self):
+        res = []
+        for rec in self:
+            res.append((rec.id, ("%s" % rec.value_answer_id.name if rec.question_type == "simple_choice" else rec.value_text_box)))
```

Comment:
Linting: would make a list comprehension + use ``reg`` (as registration) + multi-line to ease reading,like
```
return [
    (reg.id,
     reg.value_answer_id.name if reg.question_type == "simple_choice" else reg.value_text_box
    )
    for reg in self
]
```